src/main/java/com/faforever/client/replay/Replay.java

```diff
@@ -219,6 +222,19 @@ public void setEndTime(Temporal endTime) {
     return endTime;
   }
 
+  @Nullable
+  public Integer getReplayTicks() {
+    return replayTicks.get();
+  }
+
+  public void setReplayTicks(Integer replayTicks) {
+    this.replayTicks.set(replayTicks);
+  }
+
+  public ObjectProperty<Integer> replayTicksProperty() {
```

Comment:
just for my information, where is that object property used?

Reply:
I didn't use it anywhere. I just included it because the other attributes seemed to have them. IDK what an ObjectProperty even does.

Reply:
@axel1200 ? :-)

Reply:
Yes u do it correctly

Reply:
it is an object where u can listen for changes

Reply:
It's an observable object, following the observer pattern from software engineering. You can add a listener that gets notified on changes. The property, which is introduced by javafc id a special implementation of this observer pattern allowing properties to be bound to others (supporting converters) therefore updating each other/ UI components when changed.